### ISSUE TITLE
No version match found for npm:react@~0.14.0

### ISSUE THREAD

**Issue by stooboo** [gaearon/react-dnd#162]
Hi

When installing in jspm using

   jspm install npm:react-dnd --save

I get 

err No version match found for npm:react@~0.14.0

the package.json is specifying 
   "react": ">=0.13.0 <0.15.0"

so I'm going to go and look at the other 'dependencies' .. dnd-core etc. 

Just thought you should know ;-)
(hopefully I'm not missing something totally obvious ! )

cheers
Stu

**Comment by gaearon** [comment]
Are you using it with React 0.14 alpha?

**Comment by stooboo** [comment]
Thanks that was quick ..

 I 'thought' I was using 0.13.3. but checking now...  jspm IS using 0.14 .. :-(

I'm just trying to get it to 'roll back' 

I'll update this asap

DOH ! I suspected it was something I was doing wrong 

thanks for responding so quickly 
cheers
Stu

**Comment by stooboo** [comment]
Hi

I reverted my project back to react:@0.13.3
.. tried installing (jspm install npm:react-dnd --save) - same error

I started a new 'project' in a clean folder with 'jspm init' etc. etc.
.. tried installing (jspm install npm:react-dnd --save) - again same error

In my 'original' project i added an overrides section to my package.json
so..
{
  "jspm": {
   ...
   "overrides": {
      "npm:react-dnd@1.0.0": {
        "peerDependencies": {
          "react": "0.13.3"
        }
      }
    }
}

etc. 
 tried installing (jspm install npm:react-dnd --save) - and it WORKS :-)

I had already tried npm clean cache etc. etc. but no joy until I added the overrides..

I'm only just beyond 'novice' with jspm so there may be a simpler/alternative answer to this 

cheers
Stu
